jung-visualization/src/main/java/edu/uci/ics/jung/visualization/renderers/CachingEdgeRenderer.java

```diff
@@ -150,13 +147,13 @@ public void stateChanged(ChangeEvent evt) {
   }
 
   //  @Override
-  public void layoutChanged(LayoutNetworkEvent<N, Point2D> evt) {
+  public void layoutChanged(LayoutNetworkEvent<N> evt) {
     N node = evt.getNode();
     Network<N, ?> network = evt.getNetwork();
     dirtyEdges.addAll(network.incidentEdges(node));
   }
 
-  public void layoutChanged(LayoutEvent<N, Point2D> evt) {
+  public void layoutChanged(LayoutEvent<N> evt) {
     System.err.println("FIX ME");

```

Comment:
Is this something that we're actually going to support?

Reply:
not sure.... It's unused now I believe.

Reply:
In which case I'm happy to excise it and we can add it back later if we need it.